libs/core/src/lib/datetime-picker/datetime-picker.component.scss

```diff
@@ -1,30 +1,25 @@
-.fd-datetime-host {
-    display: inline-block;
-    width: 230px;
+.fd-datetime {
+    display: block;
 
-    .fd-datetime {
-        display: block;
-
-        &__container {
-            display: flex;
-            align-items: center;
-            margin: 0 16px;
-        }
+    &__container {
+        display: flex;
+        align-items: center;
+        margin: 0 16px;
+    }
 
-        &__separator {
-            background-color: lightgrey;
-            width: 1px;
-            margin: 42px 28px;
-            align-self: stretch;
-        }
+    &__separator {
+        background-color: lightgrey;
+        width: 1px;
+        margin: 42px 28px;
+        align-self: stretch;
+    }
 
-        fd-popover {
-            display: block;
-        }
+    fd-popover {
+        display: block;
+    }
 
```

Comment:
Remove file due to formatting

Reply:
this is not a formatting change. removal of the fd-datetime-host class